### PR TITLE
update(tf): update distribution config to be consistent with the current state

### PR DESCRIPTION
**Pull request by LucaGuerra**
This updates terraform for distribution:
* The tf version is updated, the same we use for Prow
* I removed the three repos that were apparently created without tf as terraform refuses to touch those. I wonder if we can fix it?
* I added encryption configuration to the buckets to reflect what is already live (those configs result in no change when applied)

### CODE REVIEW

File: config/distribution/containers.tf [hunk 1]
```diff
@@ -44,57 +44,6 @@ data "http" "falco_readme" {
   url = "https://raw.githubusercontent.com/falcosecurity/falco/master/README.md"
 }
 
-resource "aws_ecrpublic_repository" "falco" {
```
[the remark this answers is not in the document]
Comment (LucaGuerra) [reply config/distribution/containers.tf]: Thank you @maxgio92 ! I have imported the resources :)

**Review by FedeDP — APPROVED** [review]
/approve

**Pull request merged**